students/KennanY/Lesson2/GridPrint

```diff
@@ -0,0 +1,42 @@
+#This project prints grids
+
+#1. This is the simplest form of printing
+def printGrid1():
+  print('+ - - - - + - - - - +')
+  print('|         |         |')
+  print('|         |         |')
+  print('|         |         |')
+  print('|         |         |')
+  print('+ - - - - + - - - - +')
+  print('|         |         |')
+  print('|         |         |')
+  print('|         |         |')
+  print('|         |         |')
+  print('+ - - - - + - - - - +')
+
+#printGrid1()
+
+#2. We can reduce redundancy.  More compact version
+def printGrid2():
+  print('+','-' *4, '+', '-' *4, '+')
+
+#printGrid2()
+
+#3. Let's change the size
+def printGrid3(size):
```

Comment:
per pep8 style guide: functions and var names should use snake_case style -> `print_grid3`